var/spack/repos/builtin/packages/r-tibble/package.py

```diff
@@ -41,3 +41,4 @@ class RTibble(RPackage):
     depends_on('r-assertthat', type=('build', 'run'))
     depends_on('r-lazyeval@0.1.10:', type=('build', 'run'))
     depends_on('r-rcpp', type=('build', 'run'))
+    depends_on('r-rlang', type=('build', 'run'))
```

Comment:
I believe this should be a versioned dependency, and r-lazyeval should be as well. It looks like one was swapped for the other in r-tibble version 1.3.1.

`depends_on('r-lazyeval@0.1.10:', type=('build', 'run'), when=@:1.3.0)`
`depends_on('r-rlang', type=('build', 'run'), when=@1.3.1:)`

Source: https://github.com/tidyverse/tibble/blob/master/NEWS.md#misc

Reply:
these two are now changed to versioned dependencies